.eslintrc

```diff
@@ -1,22 +1,29 @@
 rules:
-  indent:
-    - 2
-    - 4
-  quotes:
-    - 1
-    - single
   linebreak-style:
     - 2
     - unix
-  semi:
-    - 2
-    - always
-  comma-dangle:
-    - 1
-    - always-multiline
-  no-extra-semi:
+  no-console:
     - 1
 env:
   browser: true
   jquery: true
-extends: 'eslint:recommended'
+globals:
+  $: false
+  alert: true
+  Analytics: false
+  BugLinks: true
+  d3: false
+  DateFilters: false
+  ga: false
+  location: true
+  MG: false
+  moment: false
+  PaginationUtils: true
+  Promise: false
+  Qs: false
+  SignatureReport: false
+  socorro: false
+  window: false
```

Comment:
Setting the [environment](https://eslint.org/docs/user-guide/configuring#specifying-environments) to include `browser` should negate the need for this and a few other globals in this list.